factory/stateComponents.go

```diff
@@ -25,6 +25,7 @@ type StateComponentsFactoryArgs struct {
 	ShardCoordinator sharding.Coordinator
 	Core             CoreComponentsHolder
 	StorageService   dataRetriever.StorageService
+	IsImportDbMode   bool
```

Comment:
As discussed, let's have a type for this with 2 consts. We will use it also in some other parts

Reply:
Done.